[doc] Update install.rst and Misc. (#1009)
taichi-dev/taichi: docs/gui.rst

    Create a window.
    If ``res`` is scalar, then width will be equal to height.

>     This creates a window whose width is 1024 and height is 768:

Reviewer: I'd say ''This creates a window of 1024 x 768'' or "This creates a window whose resolution is 1024 x 768"

Author: Sounds good, thank you! Btw, please feel free to use the `suggestion` feature (item 6 in https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) of github to make us work more efficiently.

Reviewer: Sure thanks, sorry I'm not so used to the github workflow.

                gui.show(f'{frame:06d}.png')

Paint on a window
-----------------

.. function:: gui.set_image(img)

    :parameter gui: (GUI) the window object
    :parameter img: (np.array or Tensor) tensor containing the image, see notes below

    Set an image to display on the window.

>     The window pixels, ``i`` from left to right, ``j`` from bottom to top, are set to the values of ``img[i, j]``.

Reviewer 2: Thank for fixing the typo.. but I think `img[i, j]` is one pixel (one value), instead of pixels (values).
```suggestion
    The window pixel at ``i`` from left to right and ``j`` from bottom to top, is set to the value of ``img[i, j]``.
```

Reviewer: I'd say ''The image pixels are set from the values of ``img[i,j]``, where ``i`` indicates the horizontal coordinates from left to right while ``j`` the vertical coordinates from bottom to top.

    The data type of ``img`` must be one of:

    * ``float32``, range ``[0, 1]``

    * ``float64``, range ``[0, 1]``

    * ``uint8``, range ``[0, 255]``

    * ``uint16``, range ``[0, 65535]``

    * ``uint32``, range ``[0, 4294967295]``

> 

Reviewer 2: Let's say something about the #1005 - if a float value is not in range `[0, 1]`, it will be clip into that range.
```suggestion
    .. note ::
    
        If a float value is not in the range ``[0, 1]``, it will be clipped into that range, e.g.: ``1.3 -> 1``, ``-0.2 -> 0``, ``0.5 -> 0.5``.
        
```

.. function:: gui.circle(pos, color = 0xFFFFFF, radius = 1)

    :parameter gui: (GUI) the window object

>     :parameter pos: (tuple of 2) the position of the circle
>     :parameter color: (optional, RGB hex) the color to fill the circle
>     :parameter radius: (optional, scalar) the radius of the circle

Reviewer 2: Double `the` sounds strange to me... I guess we mainly want to emphersize the uniqueness of that `circle`, instead of the property:
```suggestion
    :parameter pos: (tuple of 2) position of the circle
    :parameter color: (optional, RGB hex) the color to fill the circle
    :parameter radius: (optional, scalar) radius of the circle
```

Reviewer: I second these changes. The more the articles the clearer the meaning, isn't it? Hahaha I'm just joking but here we need those for sure.

    Draw a solid circle.

.. function:: gui.circles(pos, color = 0xFFFFFF, radius = 1)

    :parameter gui: (GUI) the window object

>     :parameter pos: (np.array) the positions of the circles
>     :parameter color: (optional, RGB hex or np.array of uint32) the color(s) to fill the circles
>     :parameter radius: (optional, scalar) the radius (radii) of the circles

Reviewer 2: ```suggestion
    :parameter pos: (np.array) positions of the circles
    :parameter color: (optional, RGB hex or np.array of uint32) color(s) to fill the circles
    :parameter radius: (optional, scalar or np.array of uint32) the radius (or radii) of the circles
```
`scalar or np.array of uint32`, since you said `radii`..
Btw, do we have `radius(es)` in en? Not sure if all people know what this is..

Reviewer: No, we don't really have radiuses. Probably there is a word like that in dictionaries but I never saw it being used.

    Draw solid circles.

.. note::

>     If ``color`` is a numpy array, circle at ``pos[i]`` will be colored with ``color[i]``,
>     so in this case ``color`` must have the same number of elements as ``pos``.

Reviewer 2: ```suggestion
    If ``color`` is a numpy array, the circle at ``pos[i]`` will be colored with ``color[i]``,
    so in this case ``color`` must have the same number of elements as ``pos``.
```
All use `the circle`?

Reviewer: I'd say ''therefore they must have the same size'' or ''therefore ``color`` and ``pos`` must have the same size''.
